test/unit/autocomplete.ts

```diff
@@ -294,13 +303,13 @@ suite('Autocomplete', ({ expect, spy, stub }) => {
     it('should call flux.saytProducts() with original query', () => {
       const saytProducts = spy();
       const state = { a: 'b' };
-      const querySelector = stub(Selectors, 'autocompleteQuery').returns(query);
-      autocomplete.flux = <any>{ emit: () => null, saytProducts, store: { getState: () => state } };
+      select = Autocomplete.prototype.select = spy(() => query);
```

Comment:
if you switch `select` to be a stub, then you can do `select.returns(query)` here instead